src/MySqlConnector/MySqlDecimal.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Globalization;
+
+namespace MySqlConnector;
+
+	public struct MySqlDecimal : IComparable, IComparable<MySqlDecimal>, IConvertible, IEquatable<MySqlDecimal>
```

Comment:
This should be a `readonly struct`.

Unnecessary indentation here (and through rest of file).

Reply:
I don't think `IConvertible`, `IComparable`, `IComparable<T>` are necessary to implement; my API sketch (in https://github.com/mysql-net/MySqlConnector/issues/1070#issuecomment-952193539) didn't call for them.

Even `IEquatable<T>` could be dropped; are `2.5` and `2.500` "equal" or not? Implementing it would just add unnecessary complexity.

Reply:
Got it.
Thanks for the review